pandas/io/parsers.py

```diff
@@ -125,7 +125,8 @@
     are duplicate names in the columns.
 dtype : Type name or dict of column -> type, default None
     Data type for data or columns. E.g. {'a': np.float64, 'b': np.int32}
-    Use `str` or `object` to preserve and not interpret dtype.
+    Use `str` or `object` together with passing `keep_default_na=False` and
```

Comment:
this is too prescriptive here, rather you can put a pointer to the NA Values section

Reply:
Thanks. I made a new commit to my branch.